### DISCUSSION TITLE
Sidecar unable to make egress HTTP/HTTPS calls after upgrading to OpenShift Service Mesh 2.6.1 (Istio 1.20)

### DISCUSSION THREAD

**Discussion by somnathpathak** [istio/istio#53313] [Q&A]
We recently upgraded to OpenShift Service Mesh 2.6.1, which uses Istio 1.20 internally. Since the upgrade, we've been experiencing an issue where the Istio sidecar is unable to make egress calls, specifically HTTP/HTTPS traffic. This is leading to "connection refused" errors in our startup probes, as observed in the logs of the istio-proxy container.

**Comment by somnathpathak** [comment]
Apologies if this isn't the right approach, but since we're looking for a prompt resolution, I would appreciate it if @howardjohn could offer some guidance on this issue.

**Reply by zirain** [reply]
if you using a vendor distribution, the best way is go there for help, most of the users/maintainers here didn't have an environment to reproduce.
